### DISCUSSION TITLE
请问umi框架支持运行时配置proxy中的header么

### DISCUSSION THREAD

**Discussion by liujiajun1994** [umijs/umi#11257] [Q&A]
需要在运行时获取用户登录信息后，配置proxy中的header，保证除了fetch type的请求，也可以携带jwt信息

**Comment by fz6m** [comment]
参考这个：https://github.com/umijs/umi/issues/10760#issuecomment-1471158059

理论上应该是你的请求库在做添加 header 这件事，而不是 proxy ，因为线上就没有本地的开发 proxy 了。
